env.yml

```diff
@@ -44,7 +44,8 @@ dependencies:
   - transformers
   - tokenizers <0.13.2
   - sentencepiece
-  - biotite # required for ESM models
+  - biotite # required for ESM model
+  - pytorch_geometric >=2.4.0
```

Comment:
Which machine have you tested this on ? I have a suspicion that some module might be required (e.g `pytorch_scatter`). 

Could you make sure that this notebook ([pyg integration](https://github.com/datamol-io/molfeat/blob/main/docs/tutorials/pyg_integration.ipynb)) works with your changes and adapt it if needed ?

Reply:
I am testing on Mac m2. The tutorial runs with no issues. It seems pytorch_scatter is not necessary here.